rpc/src/v1/types/asset_output.rs

```diff
@@ -13,16 +13,20 @@
 //
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
+extern crate rustc_serialize;
 
 use cjson::uint::Uint;
 use ctypes::transaction::{AssetMintOutput as AssetMintOutputType, AssetTransferOutput as AssetTransferOutputType};
-use primitives::{Bytes, H160, H256};
+use primitives::{H160, H256};
+use rustc_serialize::hex::{FromHex, FromHexError, ToHex};
+
+use std::iter::FromIterator;
```

Comment:
Please move this line before line 18.
https://github.com/CodeChain-io/codechain/wiki/Coding-Style#imports-order

Reply:
Fixed.